### DISCUSSION TITLE
[Suggestion Feature] Phantoms Manager

### DISCUSSION THREAD

**Discussion by RSLuke** [Zrips/CMI#1439] [Enhancement]
It's kinda simple, it would consist of having commands so we can manager the phantoms spawn on players. For example I'm an admin and I'd like the phantoms to never spawn on me at night and store their sleep time so i get the full herd when i'm getting back to survival. I don't know if I made it clear.
Btw if i'm in creative/god and near a player and even if he has slept enough, my phantoms will spawn on him.

**Comment by x1p** [comment]
There are plugins for such.
Search for `NoCreativePhantoms`
There are even plugins which control the amount of phantoms. But maybe that a little too much.

**Comment by RSLuke** [comment]
I know but CMI's role is kinda to replace all essentials functions, and I think controlling the phantoms is essential.

**Comment by Zrips** [comment]
This is only possible with paperspigot, as they have special event for phantom spawning which can be canceled. Other then this, we cant really determine why phantom spawned.
